### PR TITLE
WT-4668 lint

### PR DESCRIPTION
**Pull request by keithbostic**
@bvpvamsikrishna, can you please look at c29e5c1?

Just in case we should be doing something with the `ts` assignment.

### CODE REVIEW

File: src/txn/txn_timestamp.c [hunk 1]
```diff
@@ -655,22 +655,18 @@ __wt_txn_commit_timestamp_validate(WT_SESSION_IMPL *session, const char *name,
...
+	 * If roundup timestamps is configured, the commit timestamp will be
+	 * rounded up to the prepare timestamp.
 	 */
-	if (F_ISSET(txn, WT_TXN_PREPARE) && ts < txn->prepare_timestamp) {
-		/*
-		 * If roundup timestamps is configured, the commit timestamp
-		 * will be rounded up to the prepare timestamp.
-		 */
-		if (F_ISSET(txn, WT_TXN_TS_ROUND_PREPARED))
-			ts = txn->prepare_timestamp;
```
Comment (bvpvamsikrishna) [inline src/txn/txn_timestamp.c]: @keithbostic, is not the statement ```ts = txn->prepare_timestamp``` missing in code change ?.

Comment (keithbostic) [reply src/txn/txn_timestamp.c]: Yes, that is the heart of the change. As far as I can tell, the assignment was never used.

Comment (bvpvamsikrishna) [reply src/txn/txn_timestamp.c]: @keithbostic, it is used in the caller. without that assignment, the ```ts``` holds the specified ```commit_timestamp```. with the ```WT_TXN_TS_ROUND_PREPARED``` option, if the ```commit_timestamp``` is less than ```prepare_timestamp```, commit timestamp will be rounded to the prepare timestamp

Comment (bvpvamsikrishna) [reply src/txn/txn_timestamp.c]: @keithbostic, I have not checked that ```ts``` is passed by value. the original code has a bug.

**Pull request merged**